Jenkinsfile

```
@@ -120,7 +120,7 @@ void withMavenEnv(List envVars = [], def buildType, def javaVersion, def body) {
     // Using the "tool" Workflow call automatically installs those tools on the
     // node.
     String mvntool = tool name: "mvn", type: 'hudson.tasks.Maven$MavenInstallation'
-    String jdktool = tool name: "jdk${javaVersion}", type: 'hudson.model.JDK'
+    String jdktool = tool name: "jdk8-adopt", type: 'hudson.model.JDK'
```

We should test Adopt for both Java 8 and Java 11, shouldn't we?
```suggestion
    String jdktool = tool name: "jdk${javaVersion}-adopt", type: 'hudson.model.JDK'
```

---

we already use it on 11 @MarkEWaite.

This just checks the change from oracle ~8.160 to Adopt ~2.240